test/parallel/test-stdin-require.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const execSync = require('child_process').execSync;
+const fs = require('fs');
+const path = require('path');
+
+if (common.isWindows) {
+  common.skip('Skipping test on windows');
```

Comment:
Maybe include why it's being skipped? I don't know if we usually do that or not...